### ISSUE TITLE
Usage details?

### ISSUE THREAD

**Issue by kf6kjg** [InWorldz/aperture#1]
Is this required to be one Aperture server per region (Halcyon) server operating system, or can/should there be one central Aperture server, or possibly a separate cluster of Aperture servers that are distinct from the region servers?

From what I hear from Vin it's supposed to be 1 Aperture per server, and each server has at least one copy of Halcyon loaded on it: one independent copy of Halcyon per region.  Where server in this case means an installation of an operating system - whether on a VM or directly on the hardware is irrelevant. :)

**Comment by appurist** [comment]
Yes, Aperture is normally run with one local copy per server machine / VM.  In gridmode, there may be multiple Halcyon region instances running on that machine, all sharing a single Aperture, but each machine/VM typically runs its own local Aperture server.

**Comment by kf6kjg** [comment]
That's what I understood. Good.

Do you know any reason why it couldn't be run separately?  I'm thinking of
localhost-only connections or whathaveyou.  Of course firewalls would have
to be configured, yada yada, but I'm just thinking about architecture here.

**Comment by ddaeschler** [comment]
The halcyon code base may feed out the local ip and host name when giving the location of the aperture cap to the client. You'll have to check it out. That would be easy to fix though
